frontend/src/lib/components/ResizableTable/TableConfig.tsx

```diff
@@ -20,48 +19,41 @@ interface TableConfigInterface {
     defaultColumns?: string[] // To enable resetting to default
     onColumnUpdate?: (selectedColumns: string[]) => void
     saving?: boolean // Whether the saving routine is in process (i.e. loading indicators should be shown)
-    mainActionComponent?: JSX.Element
 }
 
 export function TableConfig({
-    exportUrl,
     selectedColumns,
     availableColumns,
     onColumnUpdate,
-    mainActionComponent,
     ...props
 }: TableConfigInterface): JSX.Element {
     const { state } = useValues(tableConfigLogic)
     const { setState } = useActions(tableConfigLogic)
-
+    const doNotDisplay = false // to be resolved by issue #1534
```

Comment:
Nit: will give it a pass, but normally
1) we should try not to commit "temporary fixes" ... 
2) it should be `doNotDisplay = true` or `display = false`, not `doNotDisplay == false` and then used in reverse :)

Reply:
Ha, fair! Updated the name